Podverse/AppDelegate.swift

```diff
@@ -101,13 +101,17 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     }
 
     func skipBackwardEvent() {
-        PVMediaPlayer.shared.previousTime(seconds: 15)
-        PVMediaPlayer.shared.setPlayingInfo()
+        if let currentItem = pvMediaPlayer.avPlayer.currentItem {
+            pvMediaPlayer.goToTime(seconds: CMTimeGetSeconds(currentItem.currentTime()) - 15)
```

Comment:
Why not call setPlayingInfo() directly in the goToTime() method of the PVMediaPlayer instead? It is always being called after a goToTime from what it seems like.